_includes/disclaimer.html

```diff
@@ -1,5 +1,12 @@
-<div class="disclaimer">
+<header class="site-header">
```

Comment:
But those can be distinguished by hierarchy. My understanding of `<header>` as a semantically named tag in HTML5 is that it should not appear twice in the same level of hierarchy, which ensures it should never need a class name.